src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+
+    @Override
+    public List<Machine> get() {

```

Comment:
```suggestion
    public List<Bulldozer> get() {

```
зігдно з умвою у нас клас повинен повертати специфічний тип для кожного з продюсерів